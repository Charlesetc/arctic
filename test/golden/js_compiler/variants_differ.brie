define main [
	define x (Hi "there")
	x :- You "turtle"
]
