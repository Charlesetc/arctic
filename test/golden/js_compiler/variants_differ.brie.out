const global_functions = {1: {}};
function main() {
	var x = {Hi: ["there"]};
	return (x = {You: ["turtle"]});
}
var test/golden/js_compiler/variants_differ.brie_main = new_closure(1)
console.log(main());